src/System.Private.ServiceModel/tools/setupfiles/BridgeCertificateInstaller.sh

```diff
@@ -122,26 +135,26 @@ else
     echo "Certificate Authority will be written to '$__cafile'"
 fi 
 
-if [ -n "$__bridge_host" ]; then 
-    echo "Certificate will be obtained from BridgeHost '$__bridge_host'"
-    if [ -n "$BridgeHost" ]; then 
-        echo "WARNING: This will replace the BridgeHost specified in the environment, '$BridgeHost', for the duration of this script"
+if [ -n "$__service_host" ]; then 
+    echo "Certificate will be obtained from ServiceHost '$__service_host'"
+    if [ -n "$ServiceUri" ]; then 
+        echo "WARNING: This will replace the ServiceUri specified in the environment, '$ServiceUri', for the duration of this script"
     fi 
     
     # set BridgeHost as an external env var so that this can be picked up by the Bridge
```

Comment:
I want to change the name of ServiceUri as it doesn't contain a Uri, just a hostname. That will need to wait for another time. I'll change the comment.